gaek/ndb_json.py

```diff
@@ -50,6 +52,9 @@
 )
 
 
+DATE_RE = re.compile(r'\d{4}[-/]\d{2}[-/]\d{2}')
```

Comment:
This would probably be faster if you anchored it to the beginning of the line (with '^'). I assume that would be a reasonable thing to do.